.github/workflows/deploy-image.yml

```diff
@@ -0,0 +1,100 @@
+name: Create and publish a Docker image
+
+on:
+  push:
+    branches:
+      - 'main'
+    tags:
+      - 'dvalin-tools-event-agent:v*'
+  pull_request:
+    branches:
+      - 'main'
```

Comment:
Not sure if this action should only be triggered by tags. Otherwise it would build a new version every time something tries to merge into main (for example when the event agent picks up a new event, on new releases etc)

Reply:
Well, we might also want to have PR versions, to be able to test a new image before merging.
As for `main`... I'm not sure. Ideally, we would want to update `latest`. But maybe we could have it only if there's an update in the subdir.

To be noted that if there's no change, the resulting image should have the same hash.
Note: hmmm 🤔 actually, it might not, as it embeds a clone of the repo, for the data.

Reply:
I added this
```yaml
    paths:
      - 'dvalin-tools/**'
```
To only trigger this workflow when there are changes inside the tool dir. What do you think about it?

(Sorry it took me so long to do so little 🙇 )

Reply:
No worries. Take your time! ^^

Restriction only to tools dir should mitigate this issue.